### PR TITLE
Task[SDK-2659]: Add android 13 support

### PR DESCRIPTION
**Pull request by shivamsharma2710**
Changes:
- Upgrade the compileSDKVersion of plugin's android to use API level 33.
- Add an option to pass push primer config inside initProperties of the Signed Call init() method.

### CODE REVIEW

File: test/clevertap_signedcall_flutter_test.dart [hunk 1]
```diff
@@ -51,6 +51,12 @@ class MockCleverTapDirectcallFlutterPlatform
     // TODO: implement setDebugLevel
     throw UnimplementedError();
   }
+
+  @override
+  Future<void> disconnectSignallingSocket() {
+    // TODO: implement disconnectSignallingSocket
```
Comment (darshanclevertap) [inline test/clevertap_signedcall_flutter_test.dart]: @shivamsharma2710 what about this TODO?

Comment (shivamsharma2710) [reply test/clevertap_signedcall_flutter_test.dart]: @darshanclevertap this is the unit test file which implements the plugin methods coming from Signed Call.  It's left empty as of now. 

What do you suggest? I will try to implement before the next release 0.0.3 is due.

Comment (darshanclevertap) [reply test/clevertap_signedcall_flutter_test.dart]: @shivamsharma2710 Remove TODO and see the core Flutter SDK for how tests are written. If you can add it now then add it or we can add it later

Comment (shivamsharma2710) [reply test/clevertap_signedcall_flutter_test.dart]: @darshanclevertap Yes. Checking.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 ## CHANGE LOG
 
+### Version 0.0.2 (February 21, 2022)
+* Supports Signed Call Android SDK v0.0.2 and Signed Call iOS SDK v0.0.2.
+* Adds Push Primer support for the [Android 13 notification runtime permission](https://developer.android.com/develop/ui/views/notifications/notification-permission).
```
Comment (darshanclevertap) [inline CHANGELOG.md:5]: @shivamsharma2710 Mention that SC iOS v0.0.2 fixes the call screen UI issue

Comment (shivamsharma2710) [reply CHANGELOG.md:5]: @darshanclevertap Done. Please review it again.

**Review by darshanclevertap — APPROVED** [review]

**Pull request merged**